### ISSUE TITLE
tileGrid for TileDebug

### ISSUE THREAD

**Issue by pdxrlj** [MelihAltintas/vue3-openlayers#250]
Thank you for writing such a great library
I want to rewrite the following code into vue. Can you give me some demos?

```
new ol.layer.Tile({
          source: new ol.source.TileDebug({
              projection: projection,
              tileGrid: osmSource.getTileGrid()
          })
      })
```

**Comment by d-koppenhagen** [comment]
I wasn't aware of the `TileDebug`. I will try to implement a new component :)

**Comment by d-koppenhagen** [comment]
I added a component and released it with version 1.3.0 as well as 2.0.0:
https://vue3openlayers.netlify.app/componentsguide/sources/tiledebug/

Please checkout the latest releases:
https://github.com/MelihAltintas/vue3-openlayers/releases
